gobblin-restli/gobblin-flow-config-service/gobblin-flow-config-service-api/src/main/snapshot/org.apache.gobblin.service.flowstatuses.snapshot.json

```diff
@@ -180,6 +180,14 @@
                 "name" : "processedCount",
                 "type" : "long",
                 "doc" : "number of records processed in the last job execution"
+              }, {
+                "name" : "jobProgress",
+                "type" : "int",
+                "doc" : "job progress as a percentage (0-100)"
+              }, {
+                "name" : "estimatedMillisecondsToCompletion",
```

Comment:
Sorry for the late comment. Do you think we should have `estimatedSecondsToCompletion` instead of Milliseconds?

Reply:
the `executionStart/EndTimes` are in milliseconds, that's why I left it in the same format. I agree we won't pay attention to milliseconds but I was planning on rounding down to appropriate units of time before formatting the result to user in the jobStatus endpoint.